### DISCUSSION TITLE
本地部署了，拉的4O流进来，返到文本框  connecT ERROR

### DISCUSSION THREAD

**Discussion by sprocsw** [heshengtao/comfyui_LLM_party#103] [General]
返到文本框  connecT ERROR, 是什么原因，我在里面填写我的APIKEY和服务器地址了。
![Uploading 截屏2024-10-09 02.41.32.png…]()

**Comment by buliyang123** [comment]
同样的问题，顶一下，我用的是仙宫云

**Comment by heshengtao** [comment]
有可能是代理服务器的问题，你可以尝试一下关闭comfyui，重启代理，打开comfyui
